homeworks/03-data-structures/src/merge-sort.test.ts

```diff
@@ -0,0 +1,11 @@
+import {mergeSort} from "./merge-sort"
+
+/* - Develop the `mergeSort` function. */
+test('mergesort should sort', () => {
```

Comment:
`mergeSort should sort correctly with compare function` IMHO, would be a better test name